google-cloud-bigquery/pom.xml

```diff
@@ -0,0 +1,654 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright © 2022 Cask Data, Inc.
```

Comment:
Are the "~" part of the license?

Reply:
Yep https://github.com/data-integrations/google-cloud/blob/develop/pom.xml#L2